handler.fish

```diff
@@ -0,0 +1,21 @@
+function __fish_command_not_found_on_interactive --on-event fish_prompt
+    functions --erase __fish_command_not_found_handler
```

Comment:
Some people have custom command not found handler function, so I don't think it's best to erase it. Perhaps you could call the function something different, it should work as it's triggered by the `fish_command_not_found` event and not by name.